bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/ModuleType.java

```diff
@@ -0,0 +1,184 @@
+import org.openhab.binding.netatmo.internal.handler.capability.WeatherCapability;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.AirQualityChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.AirQualityExtChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.BatteryChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.BatteryExtChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.CameraChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.ChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.EventChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.EventPersonChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.HomeEnergyChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.HomeSecurityChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.HumidityChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.LocationChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.MeasuresChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.NoiseChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.PersonChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.PresenceChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.PressureChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.PressureExtChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.RainChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.RoomChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.SetpointChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.SignalChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.TemperatureChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.TemperatureExtChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.Therm1ChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.TimestampExtChannelHelper;
+import org.openhab.binding.netatmo.internal.handler.channelhelper.WindChannelHelper;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * This enum all handled Netatmo modules and devices along with their capabilities
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+@NonNullByDefault
+public enum ModuleType {
+    UNKNOWN(FeatureArea.NONE, null, List.of(), List.of()),
+    NABridge(FeatureArea.NONE, null, List.of(), List.of()), // API Bridge ==============================================
```

Comment:
I think this should be `NAAccount `instead of `NABridge`.  Then the text configuation would be:
`
Bridge netatmo:NAAccount:home "Netatmo Account" [clientId="", clientSecret="", username = "", password = ""]
`

Reply:
This makes sense.

Reply:
Done